engine/src/main/java/de/digitalcollections/flusswerk/engine/flow/FlowBuilder.java

```diff
@@ -28,6 +28,12 @@
 
   private Runnable cleanup;
 
+  private boolean propagateFlowIds;
+
+  public FlowBuilder() {
+    propagateFlowIds = false;
```

Comment:
```suggestion
    this.propagateFlowIds = false;
```

Wird an weiteren Stellen auch so verwendet :)